srtcore/core.cpp

```diff
@@ -11029,16 +11012,31 @@ int srt::CUDT::processConnectRequest(const sockaddr_any& addr, CPacket& packet)
             }
         }
 
-        // send back a response if connection failed or connection already existed
-        // (or the above procedure failed)
-        if (result == -1)
+        if (result == 1)
```

Comment:
What's exactly the purpose of inverting the order of underlying statements beside confusing the reviewer? If the purpose was to add an extra condition to `else`, this could be done without this reordering, too.

Reply:
Processing expected positive result first. It is also shorter, mainly only updating the epoll. Otherwise, handle the error.